flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/ExprCodeGenerator.scala

```diff
@@ -834,6 +834,11 @@ class ExprCodeGenerator(ctx: CodeGeneratorContext, nullableInput: Boolean)
           case BuiltInFunctionDefinitions.JSON_STRING =>
             new JsonStringCallGen(call).generate(ctx, operands, resultType)
 
+          case BuiltInFunctionDefinitions.PLUS_AGG =>
+            val left = operands.head
+            val right = operands(1)
```

Comment:
Didn't add this check here (runtime) for performance purpose, since this is already checked 2 times, only the `DecimalSumAggFunction` is calling this op, moreover the args are checked in `DecimalSumAggFunction`